avocado/plugins/tmtresult.py

```diff
@@ -0,0 +1,173 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2025
+# Authors: Lukas Kotek <lkotek@redhat.com>
+#          Jan Richter <jarichte@redhat.com>
+"""
+TMT output module.
+"""
+
+import datetime
+import os
+import pathlib
+import time
+
+from avocado.core.output import LOG_UI
+from avocado.core.parser import FileOrStdoutAction
+from avocado.core.plugin_interfaces import CLI, Init, Result
+from avocado.core.settings import settings
+
+# This naming requires tmt version >= 1.29
+yaml_template = """- name: /{name}
+  result: {status}
+  start-time: "{start}"
+  end-time: "{end}"
+  duration: {time}
+  data-path: {data}
+  {custom}
+"""
+
+
+class TMTResult(Result):
+
+    name = "tmt"
+    description = "TMT result support"
+
+    @staticmethod
+    def _allowed_status(status):
+        """
+        Function matches avocado statuses with tmt ones
+        """
+        status_match = {
+            "SKIP": "skip",
+            "ERROR": "error",
+            "FAIL": "fail",
+            "WARN": "warn",
+            "PASS": "pass",
+            "INTERRUPTED": "error",
+            "CANCEL": "skip",
+            "STARTED": "info",
```

Comment:
@richtja, @lkotek, wouldn't `STARTED` better map to the `pending` status?

* https://tmt.readthedocs.io/en/stable/spec/results.html